lib/clamp-link.in

```diff
@@ -290,6 +290,14 @@ do
     IFS=$'\n' read -d '' -r -a LINES < "$REAL_PATH"
     for LINE in ${LINES[@]}
     do
+      #remove the extra colons wrapping the command
```

Comment:
you mean removing the double-quotes, not colons?

Reply:
Yes, sorry for using wrong word.
Changed the title

Reply:
Could you update the comment in the code as well? thanks

Reply:
Updated